### ISSUE TITLE
The way of development

### ISSUE THREAD

**Issue by KonstantinKuklin** [AHAAAAAAA/PokemonGo-Map#213]
It is a very helpful project, thank you guys for it! BUT the way to use it from console is wrong in my opinion.
We should try to give the web interface to rule it (them all)! That's why enough to expand the console, lets do something like
http://127.0.0.1/admin
and configure in real time application.

**Comment by hhu94** [comment]
I'm sorry, could you rephrase? I don't quite understand what you're trying to say.

**Comment by Lazarix** [comment]
Instead of starting / configuring the app on the console, he wants a web control panel.

**Comment by sadbillo** [comment]
Front-End, he wants Front-End ..  on  a Webserver.. to configure everything.

EDIT: Lazarix was faster.

**Comment by DerScreever** [comment]
we are working to this. we want to provide a good frontend, where all users can use the map etc. so we need to do this.

**Comment by JZ6** [comment]
closed
